src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -2203,6 +2217,18 @@ protected String extractAuthorName(Document doc) {
                         if(DEBUG_AUTHOR_EXTRACTION && matches!=null && matches.size()>0) System.out.println("AUTHOR: address[class*=byline]");
                     }
 
+                    // http://www.nydailynews.com/newswires/sports/kershaw-wins-bellinger-hits-2-homers-dodgers-top-indians-article-1.3245615
+                    if (matches == null || matches.size() == 0){
+                        matches = doc.select("div[itemtype$=schema.org/Person]");
+                        if(DEBUG_AUTHOR_EXTRACTION && matches!=null && matches.size()>0) System.out.println("AUTHOR: div[itemtype$=schema.org/Person]");
+                    }
+
+                    // http://nation.foxnews.com/2017/06/13/sessions-inquisition-comes-empty-trump-unscathed-after-attorney-generals-testimony
+                    if (matches == null || matches.size() == 0){
+                        matches = doc.select("p > strong:contains(By)");
```

Comment:
@skyshard  The article uses pretty much plain html tags without distinguishing css.